discount/src/main/java/com/kyriba/conference/discount/domain/DiscountEntity.java

```diff
@@ -0,0 +1,32 @@
+package com.kyriba.conference.discount.domain;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+
+import static com.kyriba.conference.discount.domain.DiscountType.WITHOUT_DISCOUNT;
+
+/**
+ * @author Igor Lizura
+ */
+@Entity
+@Table(name = "discount")
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class DiscountEntity {
+    @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "seq_identity_id")
+    @SequenceGenerator(name = "seq_identity_id", sequenceName = "seq_identity_id", allocationSize = 1)
+    @Id
+    @Column(name = "id", updatable = false, nullable = false)
+    protected Long id;
+
+    @Column(name = "type")
```

Comment:
Useless annotation

Reply:
What about the case when I will rename the property?

Reply:
Answered in another comment.